tests/test_feature.py

```diff
@@ -847,78 +870,93 @@ def test_process_index(tmpdir):
     pd.testing.assert_frame_equal(df, df_cached)
 
 
+@pytest.mark.parametrize(
+    'process_func, is_mono, applies_sliding_window, feature_names',
+    [
+        (mean, False, False, 'mean'),
+        (mean_mono, True, False, 'mean'),
+        (mean_sliding_window, False, True, 'mean'),
+        (mean_sliding_window_mono, True, True, 'mean'),
+    ]
+)
 @pytest.mark.parametrize(
     'win_dur, hop_dur',
     [
-        (1, 0.5),
-        (pd.to_timedelta(1, unit='s'), None),
-        ('16000', None),
-        ('1000ms', '500ms'),
-        ('1000milliseconds', '500milliseconds'),
-        (f'{SAMPLING_RATE}', f'{SAMPLING_RATE // 2}'),
-        pytest.param(  # multiple frames, but win_dur is None
-            None, None,
-            marks=pytest.mark.xfail(raises=RuntimeError),
-        ),
+        (0.5, 0.25),
+        (0.5, 0.5),
+        (0.25, 0.5),
+        (pd.to_timedelta(1, unit='s'), pd.to_timedelta(0.5, unit='s')),
+        ('4000', '2000'),
+        ('500ms', '250ms'),
+        ('500milliseconds', '250milliseconds'),
+        (f'{SAMPLING_RATE // 2}', f'{SAMPLING_RATE // 4}'),
```

Comment:
In the tests before we also test `hop_dur=None`, did you omit it on purpose now?

Reply:
Oh right, I re-added the test.